content/docs/2.1/concepts/authentication.md

```diff
@@ -131,6 +131,22 @@ Every parameter you define in `TriggerAuthentication` definition does not need t
       name: {trigger-authencation-name} # this may define other params not defined in metadata
 ```
 
+## Authentication scopes: Namespace vs. Cluster
+
+Each `TriggerAuthentication` is defined in one namespace and can only be used by a `ScaledObject` in that same namespace. For cases where you want to share a single set of credentials between scalers in many namespaces, you can instead create a `ClusterTriggerAuthentication`. As a global object, this can be used from any namespace. To set a trigger to use a `ClusterTriggerAuthentication`, add a `kind` field to the authentication reference:
```

Comment:
Would be nice to add some details on `ClusterTriggerAuthentication` contents, ie. that it is basically the same object as TA, either describe this or adda trimmed example of CTA